src/http2/aws-http2-message.ads

```diff
@@ -105,6 +108,8 @@ package AWS.HTTP2.Message is
    function More_Frames (Self : Object) return Boolean;
    --  Returns True if some more data are available and so should be sent
 
+   function Status_Code (Self : Object) return Messages.Status_Code;
```

Comment:
Some comments needed and `Pre => Self.Is_Defined` needed. BTW, More_Frames may also need the same pre-condition?